$OpenBSD: patch-libplanner_mrp-time_c,v 1.4 2013/09/27 12:37:20 ajacoutot Exp $

Fix calendar calculation: UTC cannot have DST.
CRITICAL **: mrp_time2_add_seconds: assertion 'secs >= 0' failed

--- libplanner/mrp-time.c.orig	Fri Mar 29 11:02:43 2013
+++ libplanner/mrp-time.c	Fri Mar 29 12:56:01 2013
@@ -173,6 +173,7 @@ mrp_time_from_tm (struct tm *tm)
 	/* This is a hack. Set the timezone to UTC temporarily. */
 	old_tz = g_strdup (g_getenv ("TZ"));
 	g_setenv ("TZ", "UTC", TRUE);
+	tm->tm_isdst = 0;
 
 	t = mktime (tm);
 
